### PR TITLE
updated the plots that didn't work

### PR DESCRIPTION
**Pull request by OhadZivan**
fixed the students questions to match the TA ones

### CODE REVIEW

**Comment by jlpearso** [comment]
@WesleyTheGeolien were these changes that @OhadZivan made due to the formatter? Also @OhadZivan we will be generating the student notebooks from the root tutorials, we just have not done this day yet. So as long as the root files are modified we will have updated student books.

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 1]
```diff
@@ -757,19 +677,7 @@
...
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "f7024cc3",
-   "metadata": {
-    "execution": {}
-   },
-   "outputs": [],
-   "source": [
-    "df = df_tmp\n"
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:772]: I think this is needed to use df_tmp else df is overwritten in the exercice cells / student bit so the df / df_tmp mechanism needs to stay

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 2]
```diff
@@ -955,13 +857,11 @@
...
    "outputs": [],
    "source": [
-    "fig, ax = plt.subplots(figsize=(20, 5))\n",
-    "selected_tweet_fraction.reset_index().plot(\n",
-    "    x=\"created_at\", y=\"counts\", style=[\".-\"], ax=ax\n",
-    ")\n",
-    "ax.set_xticklabels(ax.get_xticklabels(), rotation=45, ha=\"right\")\n",
-    "ax.axvspan(*COPdates, alpha=0.3)  # gray region\n",
-    "ax.set_ylabel(\"fraction talking about hypocrisy\")\n"
+    "fig,ax=plt.subplots(figsize=(20,5))\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:860]: please reset formatting

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 3]
```diff
@@ -1023,11 +923,11 @@
    "execution_count": null,
    "id": "96542ae9",
    "metadata": {
-    "execution": {}
+    "scrolled": true
    },
    "outputs": [],
    "source": [
-    "selected_words_2 = [..., ..., ..., ..., ...]\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1030]: I think this needs to stay as is else it is not possible to iterate over `selected_words_2` in the next line

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 4]
```diff
@@ -1044,7 +944,7 @@
     "    .count()\n",
     "    .rename(\"counts\")\n",
     ")\n",
-    "selected_tweet_fraction_2 = ...\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1047]: white space is important I think

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 5]
```diff
@@ -1177,19 +1075,15 @@
...
     "\n",
-    "afinn_wl_df = pd.read_csv(\n",
-    "    pooch_load(url_afinn_wl, filename_afinn_wl),\n",
-    "    header=None,  # no column names\n",
-    "    sep=\"\\t\",  # tab sepeated\n",
-    "    names=[\"term\", \"value\"],\n",
-    ")  # new column names\n",
-    "seed = 808  # seed for sample so results are stable\n",
-    "afinn_wl_df.sample(10, random_state=seed)\n"
+    "afinn_wl_df = pd.read_csv(pooch_load(url_afinn_wl,filename_afinn_wl),\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1081]: please reset formatting

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 6]
```diff
@@ -1221,10 +1114,10 @@
    },
    "outputs": [],
    "source": [
-    "fig, ax = plt.subplots()\n",
+    "fig,ax=plt.subplots()\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1117]: whitespace improtant

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 7]
```diff
@@ -1221,10 +1114,10 @@
...
     "afinn_wl_df.value.value_counts().sort_index().plot.bar(ax=ax)\n",
-    "ax.set_xlabel(\"Finn score\")\n",
-    "ax.set_ylabel(\"dictionary counts\")\n"
+    "ax.set_xlabel('Finn score')\n",
+    "ax.set_ylabel('dictionary counts')"
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1120]: formatting important

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 8]
```diff
@@ -1431,31 +1317,19 @@
...
+    "    sel_name =sel_words[0] if len(sel_words)==1 else 'select_talk'\n",
+    "    fig,ax=plt.subplots()\n",
+    "    ax.set_xlim(-1,1)\n",
+    "    ax.set_xlabel('adjusted Finn score')\n",
+    "    ax.set_ylabel('probabilty')\n",
+    "    counts,bins=np.histogram(smalldf.loc[smalldf[sel_name],'afinn_adjusted'],bins=np.linspace(-1,1,101),density=True)\n",
+    "    ax.plot(bins[:-1],np.cumsum(counts),color='C0',label=sel_name+' tweets')\n",
+    "    counts,bins=np.histogram(smalldf.loc[~smalldf[sel_name],'afinn_adjusted'],bins=np.linspace(-1,1,101),density=True)\n",
+    "    ax.plot(bins[:-1],np.cumsum(counts),color='C1',label='non-'+sel_name+' tweets')\n",
+    "    ax.axvline(0,color=[0.7]*3,zorder=1)\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1330]: formatting important

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 9]
```diff
@@ -1577,8 +1450,8 @@
     "    num_show = 50\n",
     "    sorted_ids = np.argsort(wordfreqs)[::-1]\n",
     "\n",
-    "    fig, ax = plt.subplots(figsize=(10, 5))\n",
-    "    ax.bar(x=range(num_show), height=wordfreqs[sorted_ids][:num_show])\n",
+    "    fig,ax=plt.subplots(figsize=(10,5))\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1453]: whitespaces needed

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 10]
```diff
@@ -1587,8 +1460,9 @@
     "    ax.set_title(title_str + \" vectorizer\")\n",
     "    ax.set_ylim(0, 10 * wordfreqs[sorted_ids][int(num_show / 2)])\n",
     "\n",
-    "    ax_wc = inset_axes(ax, width=\"90%\", height=\"90%\")\n",
+    "    ax_wc = ax.inset_axes([0.2,0.2,0.6,0.6])\n",
     "    wc.generate_from_frequencies(wordfreqs)\n",
+    "    print(wc)\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1465]: please remove print statement

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 11]
```diff
@@ -1587,8 +1460,9 @@
     "    ax.set_title(title_str + \" vectorizer\")\n",
     "    ax.set_ylim(0, 10 * wordfreqs[sorted_ids][int(num_show / 2)])\n",
     "\n",
-    "    ax_wc = inset_axes(ax, width=\"90%\", height=\"90%\")\n",
+    "    ax_wc = ax.inset_axes([0.2,0.2,0.6,0.6])\n",
```
Comment (WesleyTheGeolien) [inline tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:1463]: is this reflected in the student exercice bit?

**Review by WesleyTheGeolien — CHANGES_REQUESTED** [review]

**Comment by WesleyTheGeolien** [comment]
fixed in #245

**Pull request closed without merging**